### ISSUE TITLE
[BUG]  Could not find TSBGTask flutter_background_fetch 

### ISSUE THREAD

**Issue by sparcsaswat** [transistorsoft/flutter_background_fetch#88]
**Your Environment**
* Plugin version: 0.5.4
* Platform: iOS emulator
* OS version: 13.2.2
* Device manufacturer / model: iphone
* Flutter info (`flutter info`, `flutter doctor`):
* Plugin config

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.12.13+hotfix.8, on Mac OS X 10.15.3 19D76, locale en-GB)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 11.2.1)
[✓] Android Studio (version 3.6)
[✓] VS Code (version 1.42.1)
[✓] Connected device (3 available)

! Doctor found issues in 1 category.

**To Reproduce**
I got this error when I just call  BackgroundFetch.start() .

**Debug logs**
- ios XCode logs, 
- Android: `$ adb logcat`
Exception has occurred.
PlatformException (PlatformException(2, Could not find TSBGTask flutter_background_fetch, null))
Throw of null.
**Additional context**

**Comment by christocracy** [comment]
>Platform: iOS emulator

See the [Debugging / iOS](https://github.com/transistorsoft/flutter_background_fetch#-debugging) section:

> ⚠️  At the time of writing, the new task simulator does not yet work in Simulator; Only real devices.

**Comment by brownsoo** [comment]
This issue occurs also in Real Device. (my phone is iphone 7)
Could not find TSBGTask

**Comment by christocracy** [comment]
You’ve made a mistake in the Setup.  Go back and carefully follow all the Setup steps.

**Comment by brownsoo** [comment]
well.. I made same example. [background sample](https://github.com/brownsoo/background_sample)
But I encount an error.

**Comment by christocracy** [comment]
Try installing from git, branch `ios-start`

```yaml
dependencies:
  background_fetch:
    git:
      url: https://github.com/transistorsoft/flutter_background_fetch
      ref: ios-start
```

**Comment by brownsoo** [comment]
It works! Thank you. Hope this issue to be merged.

**Comment by christocracy** [comment]
Published to `0.5.5`
